### ISSUE TITLE
Error opening database/parameters file MacOS Mojave

### ISSUE THREAD

**Issue by BellaWilson** [totovr/SimpleOpenNI#20]
### Issue Summary
I’ve just come back to a project I was working on a month or two ago, and while it was working fine back then (and I haven’t messed with the folders it exists in), any sketch that involves Simple OpenNI comes up with the following error when run. This error only comes up for Simple OpenNI sketches (I've tried several other kinect libraries and they are all working), and I don't think I've updated processing or my operating system since it was last working. 

### Processing Output (if any)
SimpleOpenNI Version 1.96
Error opening database/parameters file.
After initialization:

Could not run the sketch (Target VM failed to initialize).
For more information, read revisions.txt and Help → Troubleshooting.

### Type of Issue
- Execution error
- Help wanted

### Your System Configuration
**Operating system**: MacOS Mojave 10.14.5
**Version of Kinect**: xbox 360: 1414
**Processing version**: 3.5.3

**Comment by TroikaTronix** [comment]
Mark Coniglio creator of Isadora here. I don't know why this error is coming up for you if you haven't changed anything about your setup, except that maybe Mojave installed an update "behind your back" and it changed something. That said, you should know that within a month we'll have native plugins that offer tracking in Isadora 3. That might be a better solution in the long term.

That said, I'm curious if the experts here have an idea what went wrong with your setup.

**Comment by BellaWilson** [comment]
I figured it out! 

It was just a little thing mac does if you have icloud setup - and not a huge amount of storage left on your device. It's been auto-uploading entire folders to the cloud and completely removing them from the hard drive (without telling me of course) to free up space, so when I ran SimpleOpenNI after a month or so, the libraries were no longer physically on my computer. 

Silly error. Lots of frustration. But now I know.
